scenario/scenario_shacl.ttl

```diff
@@ -0,0 +1,247 @@
+            sh:node scenario:StructuralShape ;
+            sh:order 4 ;
+            sh:path scenario:structural ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node scenario:TrafficShape ;
+            sh:order 3 ;
+            sh:path scenario:traffic ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node scenario:ParticipantsShape ;
+            sh:order 1 ;
+            sh:path scenario:participants ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node scenario:FormatShape ;
+            sh:order 5 ;
+            sh:path scenario:format ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node georeference:GeoreferenceShape ;
+            sh:order 7 ;
+            sh:path georeference:georeference ] ;
+    sh:targetClass scenario:scenario .
+ 
+scenario:DataSourceShape a sh:NodeShape ;
+    sh:property [ skos:example "captured reality, synthetic" ;
+            sh:datatype xsd:string ;
+            sh:description "Capture type"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of source failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "source"@en ;
+            sh:order 0 ;
+            sh:path scenario:source ] ;
+    sh:targetClass scenario:DataSourceShape .
+
+scenario:EnvironmentalShape a sh:NodeShape ;
+    sh:property [ skos:example "194.01" ;
+            sh:datatype xsd:float ;
+            sh:description "Azimuth of the sun"@en ;
+            sh:message "Validation of sunAzimuth failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "sunAzimuth"@en ;
+            sh:order 2 ;
+            sh:path scenario:sunAzimuth ] ;
+    sh:targetClass scenario:Environmental .
+
+scenario:FormatShape a sh:NodeShape ;
+    sh:property [ skos:example "ASAM OpenSCENARIO" ;
+            sh:datatype xsd:string ;
+            sh:description "Format type definition"@en ;
+            sh:in ("ASAM OpenSCENARIO") ;
+            sh:maxCount 1 ;
+            sh:message "Validation of type failed!"@en ;
+            sh:name "type"@en ;
+            sh:order 0 ;
+            sh:path scenario:type ],
+        [ skos:example "1.5" ;
+            sh:datatype xsd:string ;
+            sh:description "Version of data format"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of version failed!"@en ;
+            sh:name "version"@en ;
+            sh:order 1 ;
+            sh:path scenario:version ] ;
+    sh:targetClass scenario:Format .
+
+scenario:ScenarioGeneralShape a sh:NodeShape ;
+    sh:property [ skos:example "Functional, Logical, Concrete" ;
+            sh:datatype xsd:string ;
+            sh:description "Pegasus type of scenario"@en ;
+            sh:in ("Functional" "Logical" "Concrete") ;
+            sh:maxCount 1 ;
+            sh:message "Validation of type failed!"@en ;
+            sh:name "type"@en ;
+            sh:order 0 ;
+            sh:path scenario:type ],
```

Comment:
Yeah this is also quiet confusing. We should probably renmane the https://github.com/GAIA-X4PLC-AAD/ontology-management-base/blob/Ontology_Scenario/scenario/scenario_shacl.ttl#L81C7-L89C35 into `dataFormat`. And the other type into `abstractionLevel`. I think this will then be more clear.

Reply:
for the generic creation of such shacl's I would rather include the subcategory name, i.e.
scenarioFormatType
scenarioGeneralType

Reply:
Fin changed it to dataFormat and abstractionLevel. Including the subcategory name is not necesarry as it is "scenario:dataFormat"